torchaudio/models/tacotron2.py

```diff
@@ -0,0 +1,851 @@
+        postnet_n_convolutions (int): Number of postnet convolutions.
+    """
+
+    def __init__(self, n_mels: int, postnet_embedding_dim: int,
+                 postnet_kernel_size: int, postnet_n_convolutions: int):
+        super(_Postnet, self).__init__()
+        self.convolutions = nn.ModuleList()
+
+        self.convolutions.append(
+            nn.Sequential(
+                _ConvNorm(n_mels, postnet_embedding_dim,
+                          kernel_size=postnet_kernel_size, stride=1,
+                          padding=int((postnet_kernel_size - 1) / 2),
+                          dilation=1, w_init_gain='tanh'),
+                nn.BatchNorm1d(postnet_embedding_dim))
+        )
+
+        for _ in range(1, postnet_n_convolutions - 1):
+            self.convolutions.append(
+                nn.Sequential(
+                    _ConvNorm(postnet_embedding_dim,
+                              postnet_embedding_dim,
+                              kernel_size=postnet_kernel_size, stride=1,
+                              padding=int((postnet_kernel_size - 1) / 2),
+                              dilation=1, w_init_gain='tanh'),
+                    nn.BatchNorm1d(postnet_embedding_dim))
+            )
+
+        self.convolutions.append(
+            nn.Sequential(
+                _ConvNorm(postnet_embedding_dim, n_mels,
+                          kernel_size=postnet_kernel_size, stride=1,
+                          padding=int((postnet_kernel_size - 1) / 2),
+                          dilation=1, w_init_gain='linear'),
+                nn.BatchNorm1d(n_mels))
+        )
+        self.n_convs = len(self.convolutions)
+
+    def forward(self, x: Tensor) -> Tensor:
+        r"""Pass the input through Postnet.
+
+        Args:
+            x (Tensor): The input sequence with shape (n_batch, n_mels, mel_specgram_lengths.max()).
+
+        Return:
+            x (Tensor): Tensor with shape (n_batch, n_mels, mel_specgram_lengths.max()).
+        """
+
+        i = 0
+        for conv in self.convolutions:
+            if i < self.n_convs - 1:
+                x = F.dropout(torch.tanh(conv(x)), 0.5, training=self.training)
+            else:
+                x = F.dropout(conv(x), 0.5, training=self.training)
+            i += 1
+
+        return x
+
+
+class _Encoder(nn.Module):
+    r"""Encoder Module.
+
+    Args:
+        encoder_n_convolutions (int): Number of convolution layers in the encoder.
+        encoder_embedding_dim (int): Number of embedding dimensions in the encoder.
+        encoder_kernel_size (int): The kernel size in the encoder.
+
+    Examples
+        >>> encoder = _Encoder(3, 512, 5)
+        >>> input = torch.rand(10, 20, 30)
+        >>> output = encoder(input)  # shape: (10, 30, 512)
+    """
+
+    def __init__(self,
+                 encoder_n_convolutions: int,
+                 encoder_embedding_dim: int,
+                 encoder_kernel_size: int,
+                 ) -> None:
+        super(_Encoder, self).__init__()
```

Comment:
This is Python 2 style. `super().__init__()` should suffice.

Reply:
Fixed in the new commit.
Thanks.

Reply:
Fixed in the new commit.
Thanks!